py-polars/src/cloud.rs

```diff
@@ -0,0 +1,12 @@
+use pyo3::exceptions::PyAssertionError;
+use pyo3::prelude::*;
+
+use crate::PyLazyFrame;
+
+#[pyfunction]
+pub fn assert_cloud_eligible(lf: PyLazyFrame) -> PyResult<()> {
```

Comment:
Just to be sure. This is temporary right? Eventually this will be exposed as `prepare_cloud_payload` (something named like that.)

Reply:
Yes! I set it up like this before we discussed that. When I add the sink functionality I will rewrite it.